src/main/java/lotto/domain/Lotteries.java

```diff
@@ -0,0 +1,40 @@
+package lotto.domain;
+
+import java.util.*;
+
+public class Lotteries {
+
+    public static final int LOTTO_AMOUNT = 1000;
+    public static final int LOTTO_MAX_BUY_AMOUNT = 2000000000;
+
+    private List<Lotto> lotteries;
+
+    public Lotteries(List<Lotto> lotteries) {
+        this.lotteries = lotteries;
+    }
+
+    public Lotteries(int buyAmount) {
+        int lottoCount = buyAmount/LOTTO_AMOUNT ;
+        this.lotteries = new ArrayList<>();
+        while(lottoCount-->0) {
```

Comment:
```suggestion
        while (lottoCount-- > 0) {
```
읽는 입장에서는 실수하기 좋은 문법이라고 생각됩니다.
컨벤션을 맞춰 제어문, 반복문, 뒤에선 한 칸 띄워주시고 구분하기 좋게 만드는건 어떤가요?
같은 로직을 유지한다면 가급적 for가 가독성 면에서 더 나아보입니다.

Reply:
컨벤션 유지하면서 가독성을 위해 for문으로 수정하겠습니다